pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/ReadableSequentialTestBase.java

```diff
@@ -154,9 +157,9 @@ void limitNotBeforePosition() {
     @DisplayName("Skipping more bytes than are available")
     void skipMoreThanAvailable() {
         final var stream = sequence("0123456789".getBytes(StandardCharsets.UTF_8));
-        assertThat(stream.skip(20)).isEqualTo(10);
-        assertThat(stream.hasRemaining()).isFalse();
-        assertThat(stream.remaining()).isZero();
+        assertThrows(BufferUnderflowException.class, () -> stream.skip(20));
+        assertThat(stream.hasRemaining()).isTrue();
+        assertThat(stream.remaining()).isEqualTo(10);
```

Comment:
Is this really specified anywhere? I mean, if `skip()` throws an exception, is the current stream position guaranteed to be unchanged? It should be either added to JavaDoc, or this assertion is to be removed from the test.

Reply:
That makes sense. I'll remove this assertion from the test because it's impossible to guarantee this behavior for all implementations.